xla/service/gpu/fusions/custom.cc

```diff
@@ -201,25 +199,24 @@ absl::Status CollectSliceInfo(
     const auto* param = Cast<HloParameterInstruction>(idx_op);
     const auto* offset_value = fusion_instr.operand(param->parameter_number());
 
-    if (auto* cst = DynCast<HloConstantInstruction>(offset_value)) {
+    VLOG(0) << "Offset value:" << offset_value->ToString();
```

Comment:
This should be `VLOG(2)`, or maybe `VLOG(1)` (we typically use `VLOG(2)` for noisy information, and `VLOG(1)` for high-level pass information. `VLOG(0)` shouldn't be used.).